libraries/go/webhook.go

```diff
@@ -131,7 +133,7 @@ func (wh *Webhook) Sign(msgId string, timestamp time.Time, payload []byte) (stri
 func parseTimestampHeader(timestampHeader string) (time.Time, error) {
 	timeInt, err := strconv.ParseInt(timestampHeader, 10, 64)
 	if err != nil {
-		return time.Time{}, errInvalidHeaders
+		return time.Time{}, fmt.Errorf("unable to parse timestamp header, err: %w", errors.Join(err, ErrInvalidHeaders))
```

Comment:
why don't we return `ErrRequiredHeaders` ? It would allow the consumer to do error comparison